### ISSUE TITLE
Post Type Overlap

### ISSUE THREAD

**Issue by MarkTrillium** [Trillium-Solutions/mendocino-wp-theme#41]
Site has overlapping use for some Post Types.

"**Posts**" can be categorized as News, Service Alert, Board of Directors, etc. but have limited functionality. User wanted to create a News post, created it through Posts and categorized it as News, but the article failed to display in the News section.
<img width="271" alt="Screen Shot 2022-05-16 at 3 09 52 PM" src="https://user-images.githubusercontent.com/52221348/168690675-20ab10a5-7de9-4c7d-af6b-c2a1cc445365.png">

But there are specific Post types to create these elements. As far as I can tell these are working as expected. Instructed user to create the News post using the News Post Type.
<img width="813" alt="Screen Shot 2022-05-16 at 3 12 07 PM" src="https://user-images.githubusercontent.com/52221348/168690913-3bf96950-dce8-4800-aec1-7dbedaedcec6.png">

I think we should remove the generic "Post" post type or at least remove those categories better served by their specific post types.

**Comment by ekeaton** [comment]
@MarkTrillium Yes! Posts are the default Post type for WordPress (as you know) and typically we just re-name Posts to News on new builds etc. In this case, since this was an older site that we updated, I believe I left "Posts" up when you were migrating information from their old Board Meetings page to the new Board Meetings post-type we created for them. 

The first step is to be sure everything else is moved over from "Posts" for Service Alerts, and News to their respective new Post types, and to see if they've created any new Board Meetings using "Posts" instead of the Board Meeting Post Type. After that removing "Posts" and running a few tests to be sure all the Post types are working properly sounds good to me!
